distribution/docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,181 @@
+#!/bin/bash
```

Comment:
this is the original file from https://github.com/jboss-dockerfiles/keycloak/blob/master/server/tools/docker-entrypoint.sh (updating only the JBOSS_HOME path)